testCasa.wtest

```diff
@@ -0,0 +1,139 @@
+    }
+
+    test "Es derrochona (ya que gastó 29200)" {
+        assert.that(casaDePepeYJulian.esDerrochona())
+    }
+
+    test "La compra más cara es heladera" {
+        assert.equals(heladera, casaDePepeYJulian.compraMasCara())
+    }
+
+    test "Los electrodomésticos compramos son la heladera y la plancha" {
+        assert.equals([heladera, plancha], casaDePepeYJulian.comprados(electrodomestico))
+    }
+
+    test "los muebles comprados son: la cama y nada más" {
+        assert.equals([cama], casaDePepeYJulian.comprados(mueble))
+    }
+
+    test "no hay comida comprada" {
+        assert.equals([], casaDePepeYJulian.comprados(comida))
+    }
+
+    test "no es una mala época" {
+        assert.notThat(casaDePepeYJulian.malaEpoca())
+    }
+
+    test "falta comprar tira de asado y paquete de fideos" {
+        assert.equals([tiraDeAsado, paqueteDeFideos], casaDePepeYJulian.queFaltaComprar([tiraDeAsado, paqueteDeFideos, plancha]))
+    }
+
+    test "falta comida" {
+        assert.that(casaDePepeYJulian.faltaComida())
+    }
+
+    test "las categorías compradas son electrodoméstico y mueble" {
+        assert.equals(#{electrodomestico, mueble}, casaDePepeYJulian.categoriasCompradas())
+    }
+}
+
+describe "Testeo del apartado Cuentas Bancarias, sección cuenta corriente" {
+    method initialize(){
+        cuentaCorriente.saldo(20)
+    }
+
+    test "Depósito de 1000 pesos" {
+        cuentaCorriente.depositar(1000)
+        assert.equals(1020, cuentaCorriente.saldo())
+    }
+
+    test "Extracción de 50 con saldo previo de 1020" {
+        cuentaCorriente.depositar(1000)
+        cuentaCorriente.extraer(50)
+        assert.equals(970, cuentaCorriente.saldo())
+    }
+
+    test "No se puede extraer 2000 con saldo de 20" {
+        assert.throwsExceptionWithMessage("No cuenta con suficiente saldo para realizar la extracción", { cuentaCorriente.extraer(2000) })
+    }
+}
+
+describe "Testeo del apartado Cuentas Bancarias, sección cuenta con gastos" {
+    method initialize(){
+        cuentaConGastos.costoPorOperacion(20)
+        cuentaConGastos.saldo(500)
+    }
+    test "Depósito de 1000 pesos" {
+        cuentaConGastos.depositar(1000)
+        assert.equals(1480, cuentaConGastos.saldo())
+    }
+    test "Extracción de 30 con saldo de 1480" {
+        cuentaConGastos.depositar(1000)
+        cuentaConGastos.extraer(30)
+        assert.equals(1450, cuentaConGastos.saldo())
+    }
+
+    test "No se puede depositar 2000" {
+        cuentaConGastos.depositar(1000)
+        cuentaConGastos.extraer(30)
+        assert.throwsExceptionWithMessage("No se puede depositar más de 1000 por operación", { cuentaConGastos.depositar(2000) })
+        assert.equals(1450, cuentaConGastos.saldo())
+    }
+
+    test "Extracción de 2000 con saldo de 1450" {
+        cuentaConGastos.depositar(1000)
+        cuentaConGastos.extraer(30)
+        cuentaConGastos.extraer(2000)
+        assert.equals(-550, cuentaConGastos.saldo())
+    }
```

Comment:
bien! Los 3 últimos podías usar in initialize o hacerlos todos en 1 solo test (para el parcial combiene hacer el caso de prueba sin separar en tantos para optimizar el tiempo)